flixel/FlxGame.hx

```diff
@@ -758,8 +759,6 @@ class FlxGame extends Sprite
 		}
 		FlxArrayUtil.clearArray(FlxG.swipes);
 		#end
-		
-		filters = filtersEnabled ? _filters : null;
```

Comment:
Actually I don't think we need a new function, we should just expect that users call ```setFilters``` if they change something in that array.